integration/smartbft/smartbft_block_deliverer_test.go

```diff
@@ -199,7 +186,47 @@ var _ = Describe("Smart BFT Block Deliverer", func() {
 	})
 
 	It("correct mock", func() {
+		var err error
+		channel := "testchannel1"
+
+		/* Create peer */
+		By("Create a peer and join to channel")
+		p0 := network.Peers[0]
+		peerRunner := network.PeerRunner(p0)
+		peerProcesses = ifrit.Invoke(peerRunner)
+		Eventually(peerProcesses.Ready(), network.EventuallyTimeout).Should(BeClosed())
+
+		_, err = network.PeerAdminSession(p0, commands.ChannelJoin{
+			BlockPath:  network.OutputBlockPath(channel),
+			ClientAuth: network.ClientAuthRequired,
+		})
+		Expect(err).NotTo(HaveOccurred())
+
+		nwo.WaitUntilEqualLedgerHeight(network, channel, 11, network.Peers[0])
+	})
+
+	It("block censorship", func() {
+		var err error
 		channel := "testchannel1"
+
+		for _, mock := range mocksArray {
+			mock.censorDataMode = true
+		}
+
+		/* Create peer */
+		By("Create a peer and join to channel")
+		p0 := network.Peers[0]
+		peerRunner := network.PeerRunner(p0)
+		peerRunner.Command.Env = append(peerRunner.Command.Env, "FABRIC_LOGGING_SPEC=debug")
+		peerProcesses = ifrit.Invoke(peerRunner)
+		Eventually(peerProcesses.Ready(), network.EventuallyTimeout).Should(BeClosed())
+
+		_, err = network.PeerAdminSession(p0, commands.ChannelJoin{
+			BlockPath:  network.OutputBlockPath(channel),
+			ClientAuth: network.ClientAuthRequired,
+		})
+		Expect(err).NotTo(HaveOccurred())
+
```

Comment:
insert here a check that the peer's log includes a log message that arrives from this line:
`m.logger.Warningf("Block censorship detected, block source endpoint: %s", m.fetchSources[m.blockSourceIndex])`
internal/pkg/peer/blocksprovider/bft_censorship_monitor.go L177

for example, take a look at how its done in integration/smartbft/smartbft_test.go L120
i.e.

`Eventually(ordererRunners[1].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))`

change the runner and expected message.